src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = `
+Test vertex attributes behave correctly (no crash / data leak) when accessed out of bounds
+`;
+
+/*
+Test coverage:
+
+The following will be parameterized (all combinations tested):
+
+1) Draw call indexed? (false / true)
+  - Run the draw call using an index buffer
+
+2) Draw call indirect? (false / true)
+  - Run the draw call using an indirect buffer
+
+3) Draw call parameter (vertexCount, firstVertex, indexCount, firstIndex, baseVertex, instanceCount,
+  firstInstance)
+  - The parameter which will go out of bounds. Filtered depending on if the draw call is indexed.
+
+4) Attribute type (float, vec2, vec3, vec4, mat2, mat3, mat4)
+  - The input attribute type in the vertex shader
+
+5) Error scale (1, 100, 100000000)
+  - Offset to add to the correct draw call parameter
```

Comment:
Can we have a couple more granular steps just to make sure one of them successfully crashes the GPU / reads OOB if the implementation doesn't implement RBA correctly.

Reply:
Added some more, do you think it should be more/less granular than this new list?

Reply:
Oh, I was thinking of larger steps, like 1, 4, 10^2, 10^4, 10^6 10^9 WDYT?

Reply:
Sounds good